src/fu-util.c

```diff
@@ -2870,10 +2911,18 @@ fu_util_reinstall(FuUtilPrivate *priv, gchar **values, GError **error)
 		return FALSE;
 
 	/* try to lookup/match release from client */
-	rel =
-	    fu_util_get_release_for_device_version(priv, dev, fwupd_device_get_version(dev), error);
-	if (rel == NULL)
-		return FALSE;
+	rel = fu_util_get_backup_release_for_device_version(priv,
+							    dev,
+							    fwupd_device_get_version(dev),
+							    error);
+	if (rel == NULL) {
```

Comment:
i'm still confused what bug you're working around; is it that you want the backup remote to have a higher priority?

Reply:
Does this fix the problem properly: https://github.com/fwupd/fwupd/pull/5769 -- this should mean the workaround is no longer needed